docs/data-sources/global_router.md

```diff
@@ -27,8 +27,8 @@ data "coralogix_global_router" "example_data_by_name" {
 
 ### Optional
 
-- `id` (String) The ID of the GlobalRouter.
-- `name` (String) Name of the GlobalRouter.
+- `id` (String) The ID of the GlobalRouter
+- `name` (String) Name of the GlobalRouter. Set "router_default" for picking the default router.
```

Comment:
Leaving the id/name field empty solves the problem as well, no need to explicitly set the name to `router_default`.